backend/sirius-web-spring-collaborative-forms/src/main/java/org/eclipse/sirius/web/spring/collaborative/forms/services/LinksDescriptionService.java

```diff
@@ -0,0 +1,39 @@
+/*******************************************************************************
+ * Copyright (c) 2019, 2021 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.web.spring.collaborative.forms.services;
+
+import java.util.List;
+import java.util.Objects;
+
+import org.eclipse.sirius.web.forms.description.FormDescription;
+import org.eclipse.sirius.web.spring.collaborative.forms.api.ILinksDescriptionService;
+
+/**
+ * Service used to query the links descriptions available.
+ *
+ * @author ldelaigue
+ */
+public class LinksDescriptionService implements ILinksDescriptionService {
```

Comment:
This should be replaced by an implementation of `ILinkDescriptionProvider` in applications integrating Sirius Components